### PR TITLE
Fixing link

### LINKED ISSUE #343: Developer Modules - Module 7 links broken?
Within the Developer Modules page (https://www.w3.org/WAI/curricula/developer-modules/), it looks like the links to Module 7 are slightly incorrect. 

For context, the link to Module 7 goes to https://www.w3.org/curricula/developer-modules/rich-applications rather than where I think it's meant to go: https://www.w3.org/WAI/curricula/developer-modules/rich-applications

It's not a big problem but I thought you might want to know!

Cheers,
Serena

### PR DESCRIPTION
**Pull request by daniel-montalvo**
Fixes #343

### CODE REVIEW

**Comment by shawna-slh** [comment]
Hi @daniel-montalvo 

I was surprised to see the edits to Gemfile.lock file

I'd like to learn more about that (and confirm with Steve, as relevant)

I went ahead and fixed the link, so it's live now. :-)

https://github.com/w3c/wai-curricula/pull/346 for https://github.com/w3c/wai-curricula/issues/343

**Comment by daniel-montalvo** [comment]
> Hi @daniel-montalvo
> 
> I was surprised to see the edits to Gemfile.lock file
> 
> I'd like to learn more about that (and confirm with Steve, as relevant)

Sure,

That was a dependency update that came from a security alert. The update was applied automatically to the master branch, no so to the production branch.

Happy to differ these and see what @SteveALee says on how we should approach this.

**Comment by SteveALee** [comment]
@slhenry @daniel-montalvo 
> Happy to differ these and see what @SteveALee says on how we should approach this.

This is a bit complex to be honest :( I assume this was a dependabot suggested PR? While it's a good policy to fix these when prompted there are a couple of issues in our case: 

- It should not have been applied automatically - it's usually a PR we chose to action or not.
- as it's just the .lock file when anyone runs `bundle install` or I think "bundle update" it will get wiped.
- if it affects one repo it will probably affect all as the deps are much the same via the shared Gemfile.
- each repo has it's own gemfile.lock making updates to all requiring some command line activity or a GitHub action

It think the best approach, assuming there is a patch released for the dependency that was using the broken version, is to run `bundle install` on all repos and check then Gemfile.* back in. The dependabot PRs then seem to magically vanish as it thinks the issue is fixed. If such an update is not available then we'd want to patch all the gemfile.locks :(

What probably should really happen is someone reviews the update as even if it's declared critical it might not be at all important in our case. But that's busy work.

So in conclusion, it's probably best to alert us if it happens and we make a group decision. Actually, we usually see notifications for the github dependabot alerts but I haven't this time. Unless it happened a while back?

**Comment by SteveALee** [comment]
@slhenry @daniel-montalvo

Actually in this case I already fixed in production branch as I saw the security alerts for Kramdown and others and handled them for all repo (as mentioned above)

https://github.com/w3c/wai-curricula/commit/f9d09ecb9b7f98796c91f4cd7a5ce4b71a525e53

You could have merged in production I think to get the updates. If you had been aware ;) git status might have told you I guess.

I suppose in theory we should all be aware of the possibility that the main repo branch getting updates and should then pull to merge them. But there's probably some process we should figure out for alerting of the event.

We'll get there :)

**Comment by daniel-montalvo** [comment]
> @slhenry @daniel-montalvo
> 
> Actually in this case I already fixed in production branch as I saw the security alerts for Kramdown and others and handled them for all repo (as mentioned above)
> 
> [f9d09ec](https://github.com/w3c/wai-curricula/commit/f9d09ecb9b7f98796c91f4cd7a5ce4b71a525e53)
> 
> You could have merged in production I think to get the updates. If you had been aware ;) git status might have told you I guess.

Thanks.

Yes, it would have told me if I had checked the status of the production branch, which I did not. Now I did "git status" on my local copy and it does tell me that I'm behind. I think we can then get rid of #344, given that the slash has been added in #346

> 
> I suppose in theory we should all be aware of the possibility that the main repo branch getting updates and should then pull to merge them. But there's probably some process we should figure out for alerting of the event.
> 
I know doing "git status" is a rule of thumb, this time I just did not check the production branch.

> We'll get there :)
